test/_test847.js

```diff
@@ -0,0 +1,60 @@
+if (typeof exports === 'object') {
+	var alasql = require('..');
+}
+
+const table = {
+	name: 'midnightcalls',
+	columns: [
+		{name: 'track_name', type: 'string'},
+		{name: 'author', type: 'string'},
+		{name: 'views', type: 'int'},
+	],
+};
+
+describe('Test 847 - Testing backtick call function', function () {
```

Comment:
Probably will fail with the replace of `\n`, but I can remove to ignore the extras spaces.
My idea was basically format the string as:
```
SELECT
    * 
FROM 
    table_name
WHERE
    table_name.id = 1
```

to

```
SELECT * FROM table_name WHERE table_name.id = 1
```

but it's so dangerous, because can format a field value before the insert or real processing.